tracer/test/test-applications/debugger/Directory.Build.props

```diff
@@ -1,19 +1,24 @@
 <Project>
   <PropertyGroup>
-    <!-- only run .NET Framework tests on Windows -->
     <TargetFrameworks Condition="'$(OS)' == 'Windows_NT'">net462;netcoreapp2.1;netcoreapp3.0;netcoreapp3.1;net5.0;net6.0;net7.0</TargetFrameworks>
     <TargetFrameworks Condition="'$(OS)' != 'Windows_NT'">netcoreapp2.1;netcoreapp3.0;netcoreapp3.1;net5.0;net6.0;net7.0</TargetFrameworks>
+    <RuntimeIdentifiers Condition="'$(OS)' == 'Windows_NT' AND '$(TargetFramework)' == 'net462'">win7-x86;win7-x64</RuntimeIdentifiers>
+  </PropertyGroup>
 
-    <OutputType>Exe</OutputType>
+  <PropertyGroup>
+    <SignAssembly>True</SignAssembly>
+    <AssemblyOriginatorKeyFile>$(MSBuildThisFileDirectory)..\..\..\..\Datadog.Trace.snk</AssemblyOriginatorKeyFile>
+  </PropertyGroup>
+  
+  <PropertyGroup>
     <Platforms>x64;x86</Platforms>
-    <PlatformTarget>$(Platform)</PlatformTarget>
     <LangVersion>latest</LangVersion>
-
+    <!-- <DebugType>full</DebugType> -->
```

Comment:
This change is problematic - we want to ensure we're testing against both legacy ("windows") PDBs and Portable PDBs. Ideally, we'd want to test both in all environments (similarly as we'd like to test both Optimized/Non-optimized), but for the time being, specifying `<DebugType>full</DebugType>` meant that we're testing Windows PDB on Windows and Portable PDBs on Linux, which is a reasonable compromise.

Reply:
Talked offline with @shurivich  - turns out this change is required to make the tests pass, which means that in reality `<DebugType>full</DebugType>` for some strange reason did not have any effect previously. 

I'm resolving this comment since I'm fine with leaving it as-is for now - as I mentioned, we plan to change things so that we test with both Windows PDBs and Portable PDBs (on Windows) in a future PR.